BusinessTampereTrafficMonitoring/iot_ticket/client.py

```diff
@@ -21,10 +21,10 @@ def post_car_count(self, *, device_id: str, lane: str, count: int, timestamp: fl
 
         Required keyword arguments:
         ===========================
-        device_id:  Device ID for the camera (defined in IoT Ticket)
-        lane:       Lane ID ("Data tag" name in IoT Ticket)
-        count:      Number of cars on the lane at the given moment
-        timestamp:  Unix timestamp in seconds (when the cars were counted)
+        device_id:  Device ID for the camera (defined in IoT Ticket) (String)
```

Comment:
@Andriamanitra Thanks for the review. I knew that these would be in lowercase. But for some reason, I changed it to like C because it gives a clear datatype description. Otherwise, who are non-python users may scratch their head what is str.